resource-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/resource/HadoopResourceServiceTest.java

```diff
@@ -105,7 +105,7 @@ public void setup() throws IOException, URISyntaxException {
         writeFile(fs, dir, FILE_NAME_VALUE_00001, FORMAT_VALUE, TYPE_VALUE);
         writeFile(fs, dir, FILE_NAME_VALUE_00002, FORMAT_VALUE, TYPE_VALUE);
 
-        ConnectionDetail connectionDetail = new SimpleConnectionDetail().uri("data-service");
+        ConnectionDetail connectionDetail = new SimpleConnectionDetail().serviceName("data-service");
```

Comment:
```suggestion
        ConnectionDetail connectionDetail = new SimpleConnectionDetail().serviceName("data-service-mock");
```
Consistent naming convention